flink-runtime/src/main/java/org/apache/flink/runtime/query/QueryableStateClient.java

```diff
@@ -157,10 +168,16 @@ public QueryableStateClient(
 		this.lookupService = lookupService;
 		this.kvStateClient = networkClient;
 		this.executionContext = actorSystem.dispatcher();
+		this.executionConfig = new ExecutionConfig();
 
 		this.lookupService.start();
 	}
 
+	/** Gets the {@link ExecutionConfig} object. */
```

Comment:
I think we should also allow a setter, so that users can set the `ExecutionConfig` from a `DataStream`. Also, to be in line with `DataStream` we should call this `getExecutionConfig()`.